sigstore/_cli.py

```diff
@@ -277,10 +275,10 @@ def _parser() -> argparse.ArgumentParser:
     dsse_options.add_argument(
         "--predicate-type",
         metavar="TYPE",
-        choices=SUPPORTED_PREDICATE_TYPES,
+        choices=[p.value for p in PredicateType],
         type=str,
         required=True,
-        help=f"Specify a predicate type ({', '.join(SUPPORTED_PREDICATE_TYPES)})",
+        help=f"Specify a predicate type ({', '.join((p.value for p in PredicateType))})",
```

Comment:
Same comment below, I think these can be just `PredicateType` or `list(PredicateType)` as appropriate.